cloudflare/src/endpoints/workers/mod.rs

```diff
@@ -44,10 +48,11 @@ impl ApiResult for WorkersRouteIdOnly {}
 /// https://api.cloudflare.com/#worker-secrets-properties
 #[derive(Deserialize, Serialize, Debug, Clone, PartialEq, Eq)]
 pub struct WorkersSecret {
-  /// TODO these fields depend on the API and may be wrong
+  /// TODO: these fields depend on the API and may be wrong since unable to test
   pub name: String,
-  pub modified_on: String,
-  pub script_id: String,
+  pub r#type: String,
```

Comment:
what's this syntax? 😮

Reply:
type is a reserved name so my text editor recommended this. 
![image](https://user-images.githubusercontent.com/7578652/73196052-8cffe580-40f4-11ea-9dad-96be9c5abe3b.png)

Reply:
I'm not opposed to using core language features to solve problems

Reply:
I took your suggestion